pkg/controller/util.go

```diff
@@ -861,6 +868,62 @@ func MakeUploadServiceSpec(name string, pvc *v1.PersistentVolumeClaim) *v1.Servi
 	return service
 }
 
+// CreateCDIConfig creates cdi config manifest and sends to server
+func CreateCDIConfig(cdiClient clientset.Interface, name, url string) (*cdiv1.CDIConfig, error) {
```

Comment:
So I think we can have multiple urls (one for each namespace) and thus we have to either pass in a namespace and have a config per namespace or have the config in the cdi namespace (like we have here) and pass in multiple urls (maybe a map with ns/url pairs?) and store that in the config.

Reply:
There should only be one CDI config with a single url